features/step_definitions/index_page_steps.rb

```diff
@@ -0,0 +1,3 @@
+Given(/^I am on the index page$/) do
```

Comment:
I discuss this with Casper hand he said that we should keep index page because the been working on restaurant page.